app/views/lightweight_activities/edit.html.haml

```diff
@@ -18,77 +18,79 @@
         = f.text_field :background_image
-        .hint (Specify the full URL of a background image file. NOTE: This value is only used by Activity Player. For LARA runtime, a background image can be specified in the theme.)
+        .hint Specify the full URL of a background image file.
+      .field{:class => ("disabled" if @activity.runtime != 'LARA')}
+        = f.label :theme_id, 'LARA Theme'
+        = f.select :theme_id, options_from_collection_for_select(Theme.all, 'id', 'name', @activity.theme_id), { :include_blank => "None (inherit from sequence, project, or site default)" }
       - if current_user.is_admin
         .field
           = f.label :project_id, 'Project'
           = f.select :project_id, options_from_collection_for_select(Project.all, 'id', 'title', @activity.project_id), { :include_blank => true }
         .field
-          = f.label :is_official,  "Official CC Activity"
           = f.check_box :is_official
+          = f.label :is_official,  "Official CC Activity"
+          .hint Official activities appear in the STEM Resource Finder and should be vetted before being designated as such.
         .field
-          = f.label :is_locked, "Is this activity locked?"
           = f.check_box :is_locked
-          .hint ("Locked" activities may only be copied or edited by admin users, not regular authors.)
+          = f.label :is_locked, "Activity Locked"
+          .hint "Locked" activities may only be copied or edited by admin users, not regular authors.
       .field
-        = f.label :publication_status, 'Publication status'
+        = f.label :publication_status, 'Publication Status'
         = f.select :publication_status, options_for_select(LightweightActivity::PUB_STATUSES_OPTIONS, @activity.publication_status)
-      .field{:class => ("disabled" if @activity.latest_publication_portals.length > 0)}
-        = f.label :runtime, "Runtime environment"
-        = f.select :runtime, options_for_select(LightweightActivity::RUNTIME_OPTIONS, @activity.runtime), :disabled => @activity.latest_publication_portals.length > 0
-        .hint (Once the activity is published, you can't change this setting. You'll have to make a copy and change it for the copy instead.)
       .field
-        = f.label :description, 'Text for index page'
+        = f.label :description, 'Index Page Text'
         = f.text_area :description, :class => 'wysiwyg'
-      .field
-        = f.label :related, 'Related Content'
-        = f.text_area :related, :class => 'wysiwyg'
-      .field
-        = f.label :time_to_complete,  'Estimated time to complete, in minutes'
-        = f.text_field :time_to_complete
       .field
         = f.label :thumbnail_url, "Thumbnail URL"
         = f.text_field :thumbnail_url, :class=>"thumbnail_source"
-        %br
         .thumbnail_info.hint
-          Your thumbnail image should be 300px x 250px
+          Thumbnail image should be 1460 pixels wide by 800 pixels high.
```

Comment:
random observation: maybe we should change the name from `thumbnail` if this is the preferred size.